src/TTextEdit.cpp

```diff
@@ -68,26 +68,26 @@ TTextEdit::TTextEdit(TConsole* pC, QWidget* pW, TBuffer* pB, Host* pH, bool isLo
 {
     mLastClickTimer.start();
     if (pC->getType() != TConsole::CentralDebugConsole) {
-        mFontHeight = QFontMetrics(mpHost->mDisplayFont).height();
-        mFontWidth = QFontMetrics(mpHost->mDisplayFont).width(QChar('W'));
+        mFontHeight = QFontMetrics(mpHost->getDisplayFont()).height();
```

Comment:
Why not retrieve the font once, then call the functions on them. This happens a few times here :arrow_down:

Reply:
Fixed.